v12x/src/main/java/com/foursoft/harness/vec/v12x/navigations/SpecificationNavs.java

```diff
@@ -73,4 +64,69 @@ private SpecificationNavs() {
         };
     }
 
+    public static Function<HasSpecifications<VecSpecification>, Stream<VecOccurrenceOrUsage>> allOccurrenceOrUsages() {
```

Comment:
Aren't this just PartUsages?

Reply:
Stream?

Reply:
> Aren't this just PartUsages?

No, `DocumentVersion` also implements that. This method was used before for `DocumentVersion`s only but then I saw that the method is defined in the `HasSpecifications` interface so I used it instead.

> Stream?

Good catch. I used that before in another navigation method and extracted that, but I forgot to adjust the type correctly.

Reply:
@takebene: Btw. the change from `Stream<VecOccurrenceOrUsage>` to `List<VecOccurrenceOrUsage>` is a breaking change. The method already existed before but was moved within the class. This should be a minor release update then, shouldn't it?